packages/react-geo/src/components/MapView/MapView.tsx

```diff
@@ -1,23 +1,11 @@
+import { Amplify } from 'aws-amplify';
+import { fetchAuthSession } from 'aws-amplify/auth';
 import React, { forwardRef, useEffect, useMemo, useState } from 'react';
 import maplibregl from 'maplibre-gl';
 import { AmplifyMapLibreRequest } from 'maplibre-gl-js-amplify';
 import ReactMapGL from 'react-map-gl';
 import type { MapProps, MapRef, TransformRequestFunction } from 'react-map-gl';
-import { Amplify } from 'aws-amplify';
-import * as Auth from '@aws-amplify/auth';
-
-// Utility types for missing AmplifyConfig type, only includes Geo related key/values.
-// Note: these types should not be be used outside this file
-type AmplifyGeoOptions = {
```

Comment:
Where these types moved to the Geo JS category package?

Reply:
Yep, previously the shape of the Amplify configuration object was not exported by Amplify, but now they are!